Filtration/ViewModels/ItemFilterScriptViewModel.cs

```diff
@@ -241,7 +236,7 @@ private void ItemFilterBlocksOnCollectionChanged(object sender, NotifyCollection
                 }
             }
 
-            UpdateBlockModelsForView();
+            RaisePropertyChanged(nameof(ViewItemFilterBlockViewModels));
```

Comment:
Since this is a CollectionView, `ViewItemFilterBlockViewModels.Refresh()` should work and would prevent the entire control rebinding (I think). Try it and if it works we should use that instead of `RaisePropertyChanged`

Reply:
There are two different collectionviews and one uses the other's result as item source (Which I think prevents Refresh method being enough for update). One is used for visibility filtering the other is for other filters. The reason is to show expand/collapse button, view needs to know if a section has child blocks after filtering. Otherwise expand/collapse buttons are shown without any block in the section. [Here is relative code](https://github.com/ben-wallis/Filtration/pull/65/files/bdd121f48c43401724043f0b9c92fcf8bfdd30ee#diff-f943b18d3c0e63b1b64a2749d3372377R355)